addons/dataverse/models.py

```diff
@@ -125,6 +125,24 @@ def nodelogger(self):
             auth=auth
         )
 
+    def get_folders(self, path=None, folder_id=None):
+        """Get list of datasets from provided Dataverse alias"""
+
+        connection = client.connect_from_settings(self)
+        dataverse = client.get_dataverse(connection, self.dataverse_alias)
+
+        datasets = client.get_datasets(dataverse)
```

Comment:
Do we need error handling if client.get_datasets(dataverse) fails or client.get_dataverse(...) fails somehow?